concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md

```diff
@@ -21,14 +29,53 @@ The DataGrid component meets a variety of <a href="https://www.access-board.gov/
         <td>2.1.1 Keyboard (Level A)</td>
         <td><ul>
                 <li>Keyboard navigation is not supported for ColumnChooser items in 'drag' mode.</li>
+                <li>DataGrid is navigated with the Tab key in the data area.</li>
                 <li>The master-detail feature does not support keyboard navigation.</li>
                 <li>Adaptive rows cannot be expanded/collapsed using keyboard navigation.</li>
```

Comment:
The points about master detail and adaptive rows are no longer relevant (You deleted them for the TreeList)